projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor.helper.ts

```diff
@@ -157,7 +157,8 @@ export const jsonSchemaDataFormats: JsonSchemaDataType[] = [
   }
 ];
 
-export const dataTypeMap: {} = {};
+export const dataTypeMap: Record<string, any> = {};
```

Comment:
suggestions: would `unknown` instead of `any` make ts scream at us?

Reply:
yes... unknown hear mad TS mad.